com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md

```diff
@@ -7,7 +7,12 @@ This document describes the format of the mask map and detail map so that you ca
 To create a mask map:
 
 1. Open image editing software that supports channel editing (such as Adobe Photoshop).
-2. Drag your grayscale textures into their respective color channel. For information about which texture belongs in which channel, see [mask map](#MaskMap) and [detail map](#DetailMap).<br />![](Images/MaskMapAndDetailMap1.png)
+2. Import your grayscale textures onto separate layers.
```

Comment:
Have we made it clear why we have multiple grayscale textures? Wouldn't one texture on one layer would work the same?